docs/Operators.md

```diff
@@ -1563,19 +1563,33 @@ expect(node, inputs=[x, s, bias, mean, var], outputs=[y],
   specified by the 'to' argument and returns an output tensor of the same size in
   the converted type. The 'to' argument must be one of the data types specified
   in the 'DataType' enum field in the TensorProto message.
-  NOTE: Casting to and from strings is not supported yet.
+  
+  Casting from string tensor in plain (e.g., "3.14" and "1000") and scientific numeric representations
+  (e.g., "1e-5" and "1E8") to all numeric types is supported. Values will be implicitly truncated
+  based on the target tensor type's precision. For example, converting string "100.5" to an integer may
```

Comment:
I tried this in numpy, `100.5` to int will cause error. So I would suggest throw the error as well. If user really want to do so, they can convert to float first, then convert to int.

Reply:
+@wschin for discussion

Reply:
I guess users specify float-to-int conversion only when they really want (if not by rare accident).

Reply:
So to prevent such accident, let's throw error in such cases.

Reply:
No I mean this is an operator, not a programming language. In addition, what's the definition of throwing in an IR? You can have at most an undefined behavior because exception is runtime behavior which can't be controlled by IR.

Reply:
Discussed offline. Define this rare case (string representing 'float' -> int) as undefined behavior.